config/initializers/pagy.rb

```diff
@@ -1 +1,6 @@
 Pagy::DEFAULT[:size] = [1, 1, 1, 1]
+Pagy::DEFAULT[:items] = 25
```

Comment:
Yeah, it looks like these defaults are always overridden by [#per_page](https://github.com/DFE-Digital/npq-registration/pull/1223/files#diff-ea9171d7c8260bc97111ab15ee1f2d94cbffa182f79ebe32454cdade7ae9e3e2R18). If nothing was set there it'd fall back to this though.

Reply:
Plus it will be nice to see if the API is fast enough to cope with no pagination at all once the data is in place. Pretty much a guarantee it will make provider integrations much simpler, faster and safer.